.circleci/config.yml

```diff
@@ -1100,3 +1106,5 @@ workflows:
             - op-batcher-docker-publish
           context:
             - gcr-release
+
+# VS Code Extension Version: 1.5.0
```

Comment:
Could you remove this line?

Reply:
Yep. It's a great extension, why it wants to do this IDK.